tests/integration/targets/vmware_host_disk_info/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+# Test code for the vmware_guest_info module.
+# Copyright: (c) 2017, Abhijeet Kasurde <akasurde@redhat.com>
+# Copyright: (c) 2018, James E. King III (@jeking3) <jking@apache.org>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- import_role:
+    name: prepare_vmware_tests
+  vars:
+    setup_attach_host: true
+
+- when: vcsim is not defined
+  block:
+    # Testcase 0001: Get disk info for a host
+    - name: get list of disks from a single host
+      vmware_host_disk_info:
+        validate_certs: false
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        esxi_hostname: "{{ esxi_hosts[0] }}"
+      register: host_disk_info
+
```

Comment:
By using assert, the integration test can check to exact the results.
For example, here's like.

```yaml
    - name: Make sure if the properties defined of ESXi disk
      assert:
        that:
          - item.canonical_name is defined
          - item.capacity_mb is defined
          - item.device_ctd_list is defined
          - item.device_path is defined
          - item.device_type is defined
          - item.disk_uid is defined
          - item.display_name is defined
      loop: "{{ host_disk_info.hosts_disk_facts[esxi1] }}"
```

Reply:
Added test to 18e15942e8ccc85bcb3729134172a32c7bdca295